examples/ensemble/plot_adaboost_hastie_10_2.py

```diff
@@ -121,3 +162,11 @@
 leg.get_frame().set_alpha(0.7)
 
 plt.show()
+
+# %%
+#
+# Concluding remarks
+# ---------------------------------------------------
+#
+# We observe that the performance on both train and test sets of real AdaBoost
+# is better than that of discrete AdaBoost
```

Comment:
Reading this over again, I would favor a more literal description of the results. My main reason is because very low error rate on a training set can often be an indicator of overfitting. Thus, I think stating that performance is better could potentially be confusing to some readers.

```suggestion
# We observe that the error rate for both train and test sets of real AdaBoost
# is lower than that of discrete AdaBoost.
```